subprojects/docs/src/docs/userguide/workingWithDependencies.adoc

```diff
@@ -15,73 +15,47 @@
 [[working_with_dependencies]]
 == Working with Dependencies
 
-For the examples below we have the following dependencies setup:
+Gradle provides an extensive API for navigating, inspecting and post-processing dependencies and artifacts of resolved modules. With this API at hand, you can implement more complex requirements to your build requiring a "runtime view" of dependencies.
 
-++++
-<sample id="configurationHandlingSetup" dir="userguide/artifacts/configurationHandling" title="Configuration.copy">
-            <sourcefile file="build.gradle" snippet="setup"/>
-        </sample>
-++++
-
-The dependencies have the following transitive dependencies:
-
-shark-1.0 -&gt; seal-2.0, tuna-1.0
+=== Iterating over dependencies assigned to a configuration
 
-orca-1.0 -&gt; seal-1.0
+The <<sec:listing_dependencies,dependencies report>> is a built-in task that comes with every project independent of the applied plugins. Sometimes you'll want to implement logic based on the resolved dependencies of a project e.g. for building a plugin that visualizes the dependency graph as an image. The api:org.gradle.api.artifacts.Configuration[] API provides a wealth of methods to navigate and inspect dependencies assigned to a configuration.
```

Comment:
I see how this relates to the _dependency report_ as you can use the API for similar things. But as the first sentences (_The dependencies report is a built-in task that comes with every project independent of the applied plugins._), it feels out of place to me. Maybe just remove it?
Or add it as a not at the end of the section below the example. Would also give an opportunity to mention build scans. Something like:
_Note that if you only want to analyses dependencies of your build, you do not need to write any custom script code. You can use build scans(LINK) or the built-in dependencies(LINK) task._

Reply:
I'll remove it for now. In the beginning I tried to pin back to the content in "Inspecting Dependencies" but later I decided to keep the content in "Working with Dependencies".